portfolio.html

```diff
@@ -52,31 +52,57 @@
 							
 				</section>
 		
-				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#B4535A"></i> Projects</div>
+				<section class="projects"><div class="contentHeadings"><i class="fas fa-rocket fa-1.2x" style="color:#CE3B45"></i> Projects</div>
```

Comment:
make contentHeadings into contentHeading, more semantic when it represents a singular